### DISCUSSION TITLE
unknown command

### DISCUSSION THREAD

**Discussion by Bensal1999** [Desuuuu/klipper#29] [Q&A]
[klippy (2).log](https://github.com/Desuuuu/klipper/files/6208076/klippy.2.log)

Dear Desuuuu
I have done so many times in so many ways the installation and every time I get this massage:
![error](https://user-images.githubusercontent.com/81255199/112549453-20e34500-8dbe-11eb-8560-c3806b4739d2.png)

Everything works perfekt i can controll the printer via webinterface until I add this [t5uid1]
firmware: dgus_reloaded to the printer.cfg. This is why I think there is an issue. I have flashed the screen firmware and the klipper Logo is shown.
Maybe you could help me I would really appreciated that I am about to give up...

look at this line in the log there is the problem.  
File "/home/pi/klipper/klippy/extras/t5uid1/t5uid1.py", line 341, in _build_config
    "t5uid1_ping oid=%c", cq=cmd_queue)
  File "/home/pi/klipper/klippy/mcu.py", line 685, in lookup_command
    return CommandWrapper(self._serial, msgformat, cq)
  File "/home/pi/klipper/klippy/mcu.py", line 392, in __init__
    self._cmd = serial.get_msgparser().lookup_command(msgformat)
  File "/home/pi/klipper/klippy/msgproto.py", line 300, in lookup_command
    raise error("Unknown command: %s" % (msgname,))
error: Unknown command: t5uid1_ping
Build file /home/pi/klipper/klippy/../.config(2039): Thu Mar 25 18:50:50 2021

**Comment by Desuuuu** [comment]
The firmware currently running on your board is not the correct one or is misconfigured.

You need to configure and compile the board firmware using this repository before flashing it to your printer.
